src/Db/Table.php

```diff
@@ -384,19 +384,71 @@ public function renameColumn(string $oldName, string $newName)
         return $this;
     }
 
+    /**
+     * Update a table column, preserving unspecified attributes.
+     *
+     * This is the recommended method for modifying columns as it automatically
+     * preserves existing column attributes (default, null, limit, etc.) unless
+     * explicitly overridden.
+     *
+     * @param string $columnName Column Name
+     * @param string|\Migrations\Db\Table\Column|\Migrations\Db\Literal|null $newColumnType New Column Type (pass null to preserve existing type)
+     * @param array<string, mixed> $options Options
+     * @return $this
+     */
+    public function updateColumn(string $columnName, string|Column|Literal|null $newColumnType, array $options = [])
+    {
+        // Force preservation for updateColumn
+        $options['preserve_unspecified'] = true;
```

Comment:
```suggestion
        $options['preserveUnspecified'] = true;
```

Most keys are camelBacked in migrations.